roqoqo-qryd/src/tweezer_devices.rs

```diff
@@ -136,22 +136,22 @@ impl From<TweezerLayoutInfo> for TweezerLayoutInfoSerialize {
         let tweezer_single_qubit_gate_times: Vec<(String, SingleTweezerTimes)> = info
             .tweezer_single_qubit_gate_times
             .into_iter()
-            .map(|(k, v)| (k, v.into_iter().map(|(i_k, i_v)| (i_k, i_v)).collect()))
+            .map(|(k, v)| (k, v.into_iter().collect()))
             .collect();
         let tweezer_two_qubit_gate_times: Vec<(String, TwoTweezersTimes)> = info
             .tweezer_two_qubit_gate_times
             .into_iter()
-            .map(|(k, v)| (k, v.into_iter().map(|(i_k, i_v)| (i_k, i_v)).collect()))
+            .map(|(k, v)| (k, v.into_iter().collect()))
             .collect();
         let tweezer_three_qubit_gate_times: Vec<(String, ThreeTweezersTimes)> = info
             .tweezer_three_qubit_gate_times
             .into_iter()
-            .map(|(k, v)| (k, v.into_iter().map(|(i_k, i_v)| (i_k, i_v)).collect()))
+            .map(|(k, v)| (k, v.into_iter().collect()))
             .collect();
         let tweezer_multi_qubit_gate_times: Vec<(String, MultiTweezersTimes)> = info
             .tweezer_multi_qubit_gate_times
             .into_iter()
-            .map(|(k, v)| (k, v.into_iter().map(|(i_k, i_v)| (i_k, i_v)).collect()))
```

Comment:
What behaviour was this producing?

Reply:
this is just clippy, the internal map was useless